examples/hello.py

```
@@ -0,0 +1,30 @@
+        <li>Fast performance with Rust</li>
+        <li>Multiple output formats</li>
+    </ul>
+</body>
+</html>
+"""
+
+# Convert HTML to markdown
+markdown = convert_html_to_markdown(html)
+
+# Print the result
+print(markdown)
```

Ruff (0.8.2)

`print` found

Remove `print`

(T201)